mirth.rb

```diff
@@ -28,8 +28,8 @@ def add_data
 router = ActionDispatch::Routing::RouteSet.new
 
 router.draw do
-  get '/show/data', to: DailyDataController.action(:show_data)
-  post '/add/data', to: DailyDataController.action(:add_data)
+  match '/show/data', to: 'daily_data#show_data', via: :get
+  match '/add/data', to: 'daily_data#add_data', via: :post
```

Comment:
I pointed you at [`#match`](https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Base.html#method-i-match) because that’s where the documentation for the shorthand syntax lives, but you can keep using `get` and `post` (which, of course, don’t need the `via:` argument).